python.d/traefik.chart.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+# Description: traefik netdata python.d module
+# Author: Alexandre Menezes (@ale_menezes)
+
+from json import loads
+from base import UrlService
```

Comment:
please do
```python
from bases.FrameworkServices.UrlService import UrlService
```

[base.py](https://github.com/firehol/netdata/blob/master/python.d/python_modules/base.py) will be removed in future releases